app/tailwind.config.js

```diff
@@ -5,11 +5,60 @@ module.exports = {
   darkMode: 'class',
   // Default theme: https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L7
   theme: {
-    extend: {
-      colors: {
-        primary: colors.indigo, // primary theme color
-        secondary: colors.yellow, // secondary theme color
-      },
+    fontFamily: {
+      sans: ['"Source Code Pro"', 'monospace'],
+    },
+    fontSize: {
+      sm: ['100%', '170%'],
+      md: ['110%', '170%'],
+      lg: ['120%', '170%'],
+      xl: ['130%', '170%'],
+      xxl: ['140%', '170%'],
+      headline: ['150%', '170%'],
+    },
+    colors: {
+      transparent: 'transparent',
+      white: '#ffffff',
+      teal: '#02E2AC',
+      pink: '#F3587D',
+      'grey-100': '#E2E0E7',
+      'grey-200': '#C4C1CF',
+      'grey-300': '#A7A2B6',
+      'grey-400': '#757087',
+      'grey-500': '#0E0333',
+    },
+    stroke: {
+      transparent: 'transparent',
+      white: '#ffffff',
+      teal: '#02E2AC',
+      pink: '#F3587D',
+      'grey-100': '#E2E0E7',
+      'grey-200': '#C4C1CF',
+      'grey-300': '#A7A2B6',
+      'grey-400': '#757087',
+      'grey-500': '#0E0333',
+    },
+    strokeWidth: {
+      1: '1',
+      2: '2',
+      3: '3',
+      4: '4',
+    },
+    boxShadow: {
+      light: '0px 0px 40px 0px rgba(14, 3, 51, .1)',
+      none: 'none',
+    },
+    cursor: {
+      auto: 'auto',
+      default: 'default',
+      pointer: 'pointer',
```

Comment:
Some of the `cursor` and `colors` defined in this config are equivalent to the Tailwind defaults. Are we intentionally redefining them here for explicitness, or should they be removed for brevity / clarity over what exactly is customized/changed from the defaults?

Reply:
i just added a specific cursor (  for the dropzone ) .  thats how tailwind told to do - i assume an extend could also work but unsure ( cant test from vacation )